src/Forms/FormInput.js

```diff
@@ -23,11 +23,13 @@ FormInput.displayName = 'FormInput';
 
 FormInput.propTypes = {
     className: PropTypes.string,
+    disabled: PropTypes.bool,
+    readOnly: PropTypes.bool,
```

Comment:
The `readOnly` prop is missing a prop description.  The `disabled` prop is okay because there is a default set for it already.  The `readOnly` might actually be a good candidate for another default.